ctapipe/image/geometry_converter_hex.py

```diff
@@ -240,7 +238,6 @@ def reskew_hex_pixel_from_orthogonal_edges(x_edges, y_edges, square_mask):
     return unrot_x, unrot_y
 
 
-@jit
 def get_orthogonal_grid_edges(pix_x, pix_y, scale_aspect=True):
```

Comment:
This function is now fully jit-able with `@njit`, but when enabled, the number of bins at the end is off by 1, which seems to be to be due to a slight difference in rounding error somewhere (didn't have time to debug). For now, I've disabled the jit, but clearly this is something that will pop up at some point.